packages/tasks/src/tasks/any-to-any/data.ts

```diff
@@ -0,0 +1,55 @@
+import type { TaskDataCustom } from "../index.js";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "All-modality alignment dataset with fine-grained preference annotations and language feedback.",
+			id: "PKU-Alignment/align-anything-400k",
+		},
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "distractedboyfriendmeme.jpeg",
+				type: "img",
+			},
+			{
+				label: "Text Prompt",
+				content: "What is the colour of the shirt the man is wearing?",
+				type: "text",
+			},
+		],
+		outputs: [
```

Comment:
this is image-text-to-text schema (vision language models)
any-to-any models take 2+ inputs and 2+ outputs